pages/docs/components/overlay/popover.mdx

```diff
@@ -48,8 +48,10 @@ import {
 ## Basic Usage
 
 When using this component, ensure the children passed to `PopoverTrigger` is
-focusable. Users can tab to it using their keyboard, and it can take a `ref`. It
-is critical for accessiblity.
+focusable and has a forwarded ref. Users can tab to it using their keyboard and
+is critical for accessibility. If you are using a custom component inside
+`PopoverTrigger` (or [PopoverAnchor](#popover-anchor)) it needs Chakra's
+[forwardRef](https://chakra-ui.com/docs/components/recipes/as-prop#option-1-using-forwardref-from-chakra-uireact).
```

Comment:
```suggestion
[forwardRef](../recipes/as-prop#option-1-using-forwardref-from-chakra-uireact).
```

I think it's better to use a relative URL here instead of an absolute one, because we use subdomains like v0, v1, next, and also preview deployments as well.